template/default/layout/yaml/_constant.erb

```diff
@@ -0,0 +1,8 @@
+  - uid: <%= constant.path %>
+    fullName: <%= constant.path %>
+    langs:
+      - ruby
+    module: <%= ENV["CLOUDRAD_GEM_NAME"] %>
+    id: <%= constant.name %>
+    summary: "<%= constant.docstring.to_str.chomp.sub("--- ", "").sub("|-\n", "").gsub '"', "'" %>"
```

Comment:
Consider using a helper method for this kind of logic, instead of inlining it in the template. (You can put it in setup.rb because that's where the binding is coming from.)